src/main/java/locustTask/AbstractYesNoApiTask.java

```diff
@@ -0,0 +1,36 @@
+package locustTask;
+
+import com.github.myzhan.locust4j.AbstractTask;
+import com.github.myzhan.locust4j.Locust;
+import io.restassured.RestAssured;
+import io.restassured.http.Method;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.junit.Assert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class AbstractYesNoApiTask extends AbstractTask {
```

Comment:
With this class we avoid to rewrite the code of the execution but, what happens if we want to do another different reuqest with different baseURI? We will have to create another Abstract class to extends from locust4j and create the locustTask ... Now we will have to create one more classs ...